File: benchmarks/vm/01_fib.py
import time
def fib(n):
    if n <= 1: return n
    return fib(n-1) + fib(n-2)
t0 = time.monotonic()
r = fib(25)
print(f"fib(25) = {r}")
print(f"BENCH_RESULT: {int((time.monotonic() - t0) * 1000)} ms")

File: benchmarks/vm/01_fib.lua
local function fib(n)
    if n <= 1 then return n end
    return fib(n - 1) + fib(n - 2)
end

local t0 = os.clock()
local r = fib(25)
print("fib(25) = " .. tostring(r))
print("BENCH_RESULT: " .. tostring(math.floor((os.clock() - t0) * 1000)) .. " ms")
